apps/test/unit/templates/referenceGuides/NavigationBarTest.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import {isolateComponent} from 'isolate-react';
+import {expect} from '../../../util/reconfiguredChai';
+import NavigationBar from '@cdo/apps/templates/referenceGuides/NavigationBar';
+
+describe('NavigationBar', () => {
+  it('nav bar renders items', () => {
+    const categories = [
+      {key: 'key1', name: 'category 1', content: [], color: 'blue'},
```

Comment:
I'm noticing that content is blank in all of these tests. Wondering your thoughts on not testing that part of the implementation.